packages/botonic-plugin-ner/src/utils/environment-utils.ts

```diff
@@ -0,0 +1,24 @@
+import { Locale } from '@botonic/nlp/dist/types'
+import { loadLayersModel } from '@tensorflow/tfjs'
+import axios from 'axios'
+
+import {
+  ASSETS_DIR,
+  CONFIG_FILENAME,
+  MODEL_FILENAME,
+  MODELS_DIR,
+  NER_DIR,
+} from '../constants'
+import { ModelInfo } from '../types'
+
+export async function getModelInfo(locale: Locale): Promise<ModelInfo> {
+  const isProd = process.env.STATIC_URL !== undefined
```

Comment:
I guess isProd is not used to differenciate production & staging, but local/dev or deployed, right?
if so, I would have a temporary named st like isBrowser, or localModel